.github/workflows/development-environment.yml

```diff
@@ -89,8 +94,11 @@ jobs:
           HOMEBREW_NO_AUTO_UPDATE=1 brew install pyenv
 
       # Only after we source ~/.zprofile that the right Python will be selected
+      # XXX: We hack the .python-version since installing 3.8.13 would require a complete
+      # brew update and that will make this job take 10 more minutes
       - name: Set up pyenv
         run: |
+          echo "3.8.12" > .python-version
```

Comment:
Because I don't want CI to be even slower.

One day the runner's brew installation will be a bit more up to date.